### ISSUE TITLE
False positive for rxjs-angular/prefer-takeuntil

### ISSUE THREAD

**Issue by vladkasianenko** [cartant/eslint-plugin-rxjs-angular#14]
``` typescript

private readonly _destroy$ = new Subject<void>();

myLuckyNumber$ = this.service.getNumber().pipe(takeUntil(this._destroy$));

ngOnInit(): void {
  this.myLuckyNumber$.subscribe(); // Forbids calling `subscribe` without an accompanying `takeUntil`  rxjs-angular/prefer-takeuntil
}

ngOnDestroy() {
   this._destroy$.next();
   this._destroy$.unsubscribe();
}

```
